mod-orders/examples/composite_purchase_orders.sample

```diff
@@ -55,7 +55,9 @@
             "quantityElectronic": 1,
             "poLineEstimatedPrice": 75.47
           },
-          "orderFormat": "P/E Mix"
+          "isPackage": false,
+          "orderFormat": "P/E Mix",
```

Comment:
source is a required field but it is missing

Reply:
See comment below.